### DISCUSSION TITLE
Autowiring SsmClient not working

### DISCUSSION THREAD

**Discussion by fkjaekel** [awspring/spring-cloud-aws#1180] [Q&A]
According to the documentation, a SsmClient should automatically created, but I'm getting _No qualifying bean of type software.amazon.awssdk.services.ssm.SsmClient_

Could you tell me in what class is the @bean method for the SsmClient located so I can debug it? I only found it for other services like S3 and SQS.

I'm using version 3.1.1

**Comment by maciejwalkowiak** [comment]
This is unfortunately not as easy as looking for a `@Bean` method. Check `ParameterStoreConfigDataLocationResolver` as this is where these beans are created.
